x11-wm/vtwm/vtwm-5.4.7-r2.ebuild

```diff
@@ -45,14 +46,14 @@ src_prepare() {
 }
 
 src_configure() {
-	xmkmf || die "xmkmf failed"
-	emake depend
+	CC="$(tc-getBUILD_CC)" LD="$(tc-getLD)" \
+		IMAKECPP="${IMAKECPP:-$(tc-getCPP)}" xmkmf -a || die "xmkmf failed"
 }
 
 src_compile() {
 	emake \
-		CC=$(tc-getCC) \
-		CCOPTIONS="${CFLAGS}" \
+		CC="$(tc-getCC)" \
```

Comment:
again, consider an array

Reply:
I didn't particularly want to touch style on mass changes, they're (almost) all setup that way sometimes with multiple blocks like that, and I just adjusted.

Reply:
To clarify it's because of of IMAKECPP, new version can handle "gcc -E" which is tc-getCPP's default (along many things users could set in CPP), but setting this with arguments will cause build failures otherwise.

No harm to old ebuilds but new ones need it, so they all gained a >=1.0.8-r1. So it's one of:
1. straight
2. revbump everything to ~arch, stabilize the 41 packages later
3. add only imake to ~arch now, give it time and revisit rest of PR later

Reply:
(oops posted this in wrong thread, well it's about the ~arch bit :)